crates/services/gas_price_service/src/v1/metadata.rs

```diff
@@ -148,20 +145,25 @@ pub fn v1_algorithm_from_metadata(
         config.decrease_range_size,
         config.block_activity_threshold.into(),
     );
-    let unrecorded_blocks = metadata
+    let unrecorded_blocks_bytes: u128 = metadata
         .unrecorded_blocks
-        .into_iter()
-        .chain(config.unrecorded_blocks.clone())
-        .collect();
+        .iter()
+        .map(|(_, size)| u128::from(*size))
+        .sum();
```

Comment:
Why `unrecorded_blocks_bytes` can't be a field of the metadata, instead of calculating it each time?

Reply:
It only gets calculated on startup. I figured that would be better than adding another field to every metadata that is stored.

Reply:
<img width="596" alt="image" src="https://github.com/user-attachments/assets/f500eba3-1e87-4d67-90fa-01f7b5a35eb0">

In the worst case, it is 481 MB per year. But since we use postcard, it will be much lower. Plus, we don't need to store data for the whole year. So I wouldn't concern about the storage=)

Reply:
We can address it in https://github.com/FuelLabs/fuel-core/issues/2454